boot/src/test/java/com.taobao.arthas.boot/DownloadUtilsTest.java

```diff
@@ -0,0 +1,40 @@
+package com.taobao.arthas.boot;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class DownloadUtilsTest {
+
+    @Test
+    public void testreadMavenReleaseVersion() {
+        Assert.assertNull(DownloadUtils.readMavenReleaseVersion(""));
+    }
+
+    @Test
+    public void testreadAllMavenVersion() {
+        Assert.assertEquals(new ArrayList<String>(), DownloadUtils.readAllMavenVersion(""));
+    }
+
+    @Test
+    public void testgetRepoUrl() {
+        Assert.assertEquals("http", DownloadUtils.getRepoUrl("https/", true));
+        Assert.assertEquals("https://maven.aliyun.com/repository/public", DownloadUtils.getRepoUrl("aliyun", false));
+    }
+
+    @Test
+    public void testreadMavenMetaData() {
+        Assert.assertNull(DownloadUtils.readMavenMetaData("", true));
+
+        Assert.assertNotNull(DownloadUtils.readMavenMetaData("center", false));
```

Comment:
More checks on the return value?

Reply:
I added a few more asserts for null and not null. However, I am unsure how to check the return value as when printed it produced this -

<?xml version="1.0" encoding="UTF-8"?>
<metadata>
  <groupId>com.taobao.arthas</groupId>
  <artifactId>arthas-packaging</artifactId>
  <versioning>
    <latest>3.1.0</latest>
    <release>3.1.0</release>
    <versions>
      <version>3.0.0-RC</version>
      <version>3.0.3</version>
      <version>3.0.4</version>
      <version>3.0.5</version>
      <version>3.1.0</version>
    </versions>
    <lastUpdated>20190213182725</lastUpdated>
  </versioning>
</metadata>